CONTRIBUTING.md

```diff
@@ -128,32 +131,17 @@ Meilisearch tools follow the [Semantic Versioning Convention](https://semver.org
 This project integrates a bot that helps us manage pull requests merging.<br>
 _[Read more about this](https://github.com/meilisearch/integration-guides/blob/main/resources/bors.md)._
 
-### Automated Changelogs <!-- omit in TOC -->
-
-This project integrates a tool to create automated changelogs.<br>
-_[Read more about this](https://github.com/meilisearch/integration-guides/blob/main/resources/release-drafter.md)._
+### Release on Github and npm
```

Comment:
Tiny details, more for later when you will write guides, it's GitHub and not Github